File: Array List/ArrayListDemo.java
import java.util.*;

class ArrayListDemo
{
	public static void main(String args[]) 
	{
		try
		{
			Scanner sc = new Scanner(System.in);

			ArrayList<Integer> AL = new ArrayList<Integer> ();

			System.out.println("How many elements ");
			int iNo = sc.nextInt();

			for(int i = 0;i < iNo;i++)
			{
				System.out.println("Enter "+(i+1)+" element");
				int iValue = sc.nextInt();
				AL.add(iValue);
			}

			LinkedList ll = new LinkedList(AL);

			System.out.println("Array List is\t:"+AL);

			System.out.println("Linked List is \t:"+ll);

			System.out.println("Size of array list is \t:"+AL.size());

			System.out.println("Enter position to delete that position element");
			int pos = sc.nextInt();

			AL.remove(pos);

			System.out.println("Array List is\t:"+AL);

			AL.set(2,100);

			System.out.println("Array List is\t:"+AL);

			System.out.println("Enter number whose first index you want");
			iNo = sc.nextInt();

			pos = AL.indexOf(iNo);

			System.out.println(pos);

			System.out.println("Enter number whose last index you want");
			iNo = sc.nextInt();

			pos = AL.lastIndexOf(iNo);

			System.out.println(pos);

			Iterator itr = AL.iterator();

			System.out.println("Array List is\t:"+AL);
			while(itr.hasNext())
			{
				System.out.println(itr.next());
			}
		}
		catch(Exception e)
		{
			System.out.println(e);
		}	
	}
}

File: Array List/ArrayListDemo.cs
using System;
using System.Collections;
using System.Collections.Generic;
class ArrayListDemo
{
	public static void Main(string[] args)
	{
		try
		{
			ArrayList AL = new ArrayList();

			Console.WriteLine("How many elements");
			int iNo = int.Parse(Console.ReadLine());

			for(int i = 0;i < iNo;i++)
			{
				Console.WriteLine("Enter {0} number",i+1);
				int iNum = int.Parse(Console.ReadLine());
				AL.Add(iNum);
			}

			foreach(int i in AL)
			{
				Console.Write(i+"\t");
			}
			Console.WriteLine();

			Console.WriteLine("Enter Element to remove");
			iNo = int.Parse(Console.ReadLine());

			AL.Remove(iNo);

			foreach(int i in AL)
			{
				Console.Write(i+"\t");
			}
			Console.WriteLine();

			Console.WriteLine("Enter number whose first index you want");
			iNo = int.Parse(Console.ReadLine());

			int Pos = AL.IndexOf(iNo);

			Console.WriteLine(Pos);

			Console.WriteLine("Enter Element whose last index you want");
			iNo = int.Parse(Console.ReadLine());

			Pos = AL.LastIndexOf(iNo);

			Console.WriteLine(Pos);
			
		}
		catch(Exception e)
		{
			Console.WriteLine(e);
		}
	}
}
